### PR TITLE
add spectral_analysis_benchmarks

### PR DESCRIPTION
**Pull request by yuwunwong**

### CODE REVIEW

File: benchmarks/spectral_analysis.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    SpectrumDatasetOnOffStacker,
+    FluxPointsEstimator,
+    FluxPointsDataset,
+)
+
+N_OBS = 10
+OBS_ID = 23523
+
+def run_benchmark():
+    # Set up data store and select N_OBS times the observation OBS_ID
```
Comment (cdeil) [inline benchmarks/spectral_analysis.py]: Please use the usual Python code indentation of 4 spaces, never tabs.

File: benchmarks/spectral_analysis.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+import regions
+import astropy.units as u
+from astropy.coordinates import SkyCoord, Angle
+from regions import CircleSkyRegion
+from gammapy.maps import Map
+from gammapy.utils.fitting import Fit
+from gammapy.data import ObservationStats, ObservationSummary, DataStore
+from gammapy.background import ReflectedRegionsBackgroundEstimator
+from gammapy.spectrum.models import PowerLaw
+from gammapy.spectrum.crab import MeyerCrabModel
```
Comment (cdeil) [inline benchmarks/spectral_analysis.py]: You can remove the MeyerCrabModel import

Comment (yuwunwong) [reply benchmarks/spectral_analysis.py]: done

**Comment by yuwunwong** [comment]
please review again

**Pull request merged**